tests/processors/base/helpers/test_helpers.py

```diff
@@ -125,3 +125,55 @@ class TestSafePrepend(object):
     ])
     def test_safe_prepend(self, prepend_string, string, expected):
         assert helpers.safe_prepend(prepend_string, string) == expected
+
+class TestLocationNormalizer(object):
+    @pytest.mark.parametrize("test_input,expected", [
+        # Locations normalized by ISO-3166 name standards
+        ("China", "China"),
+        ("Brazil", "Brazil"),
+        ("United States", "United States"),
+        ("Japan", "Japan"),
+        ("Afghanistan", "Afghanistan"),
+        ("Colombia", "Colombia"),
+        ("Germany", "Germany"),
+        ("Kazakhstan", "Kazakhstan"),
+        ("Mozambique", "Mozambique"),
+        ("United Arab Emirates", "United Arab Emirates"),
+        ("United States of America","United States"),
+
+        # Locations normalized by ISO-3166 acronym standards
+        ("US","United States"),
+        ("ZMB","Zambia"),
+        ("UK","United Kingdom"),
+        ("UY","Uruguay"),
+        ("VAT","Holy See"),
+        ("YE","Yemen"),
+        ("KOR","Korea, Republic of"),
+        ("PE","Peru"),
+        ("NGA","Nigeria"),
+        ("ML","Mali"),
+
+        # Locations normalized by Levenshtein distance (name)
+        ("Chnia","China"),
+        ("Thailland","Thailand"),
+        ("The Netherlands","Netherlands"),
+        ("Vietnam","Viet Nam"),
+        ("Cina","China"),
+        ("Chiina","China"),
+        ("nhina","China"),
+        ("Cote Divoire","Côte d'Ivoire"),
+        ("thauland","Thailand"),
+        ("Thaialnd","Thailand"),
+        ("Virgin Islands (U.S.)","Virgin Islands, U.S."),
+
+        # Locations not normalized (corner cases)
+        ("Global trial(North America)","Global trial(North America)"),
+        ("Asia(except Japan)","Asia(except Japan)"),
+        ("Europe","Europe"),
+        ("Jiangsu","Jiangsu"),
+        ("Tianjin","Tianjin"),
+        ("Multinational","Multinational"),
+        ("Outside","Outside")])
```

Comment:
We don't need so many examples of the same categories. Just keep a test of each and add some corner cases (e.g. what happens if it receives `None`?)